app/controllers/api/results_controller.rb

```diff
@@ -1,3 +1,33 @@
+class Array
```

Comment:
[I pulled out it](https://github.com/ManageIQ/more_core_extensions/pull/67) because it is used in report generator. In this PR is just for usage reason for @martinpovolny UI work.